src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoMasterSchedulerConfigDao.java

```diff
@@ -47,7 +47,7 @@ final void setMapper(DynamoDBMapper mapper) {
 
     @Override
     public List<MasterSchedulerConfig> getAllSchedulerConfig() {
-        DynamoDBScanExpression scan = new DynamoDBScanExpression();
+        DynamoDBScanExpression scan = new DynamoDBScanExpression().withConsistentRead(true);
```

Comment:
I thought we had to mark a table or specific index for consistent read before it would be possible to get a consistent read, I didn't know we could ask for it on a particular scan or query.